docs/node/migrations/zero-downtime-migrations/migration-proposal.mdx

```diff
@@ -0,0 +1,106 @@
+
+```bash
+kwil-admin migrate propose --activation-period <activationPeriod> --duration <migrationDuration> --chain-id <newChainID> --rpcserver /tmp/kwil.sock
+```
+
+:::note
+An off-chain communication is required among the validator operators to decide on the details of the migration proposal. One of the validators should submit the proposal to the network for approval. Only one proposal can be applied to the network during the lifetime of the network. If multiple validators submit the same proposal, the network will only consider the first proposal submitted.
+:::
+
+To check if the proposal is submitted successfully, we can get the list of pending migration proposals as shown below:
+
+```bash
+kwil-admin migration list --rpcserver /tmp/kwild.socket
+```
+This list will contain the proposal ID which is used by the validators to vote on the proposal.
+
+
+## Approve Migration Proposal
+
+To start the migration, the migration proposal needs the approval from at least a supermajority of the validators. The validators can submit their vote approval on the migration proposal using the below command.
+
+```bash
+kwil-admin migration approve <proposal-id>
+```
+
+Operators can check the status of the proposal and the votes information as shown below:
+
+```bash
+kwil-admin migration status <proposal-id>
+```
+
+:::note
+The migration proposal will get expired if the supermajority of the validators do not approve the proposal within `100800` blocks (roughly 7 days considering the block rate of 6sec). The proposal can be resubmitted after the expiration.
+:::
+
+### Existing Network Moves Into Migration Mode
+
+Once the proposal is approved, migration is set to start after the specified activation period from the approval block. At the start of the migration, all the existing nodes will generate a snapshot of the network state at the activation height and a genesis file to be used by the new network. The snapshots and genesis files generated at the start of the migration can be downloaded as shown below:
+
+```bash
+kwil-admin migration genesis-state --output-dir </path/to/root-dir>
+```
+
+:::tip
+The command will not download any files if there is no approved migration or the migration is not yet started. Please retry the command after the migration is activated.
+:::
+
+The genesis file downloaded will have the following information set:
+- `chain-id`: Chain id of the new network.
+- `app_hash`: This field is set based on the hash of the downloaded network snapshot file at the migration start height. The data app hash field is mandatory to ensure that all the nodes in the network start with the same state using the snapshot files.
+- `validators`: This field is set to the validators from the existing network.
```

Comment:
I think we decided that this can be edited?  I'm going through a zdt migration now, and the old network has two validators, but I'm testing with a new network starting with just one validator.  So I removed the second one in this section.  That should work right?

Also, somewhere in this page there should be mention of configuring the new node with an RPC service from which it will pull the changes to replicate during the migration period between start_height and end_height.